pyanaconda/modules/common/structures/rdp.py

```diff
@@ -0,0 +1,69 @@
+#
+# DBus structure for RDP UI module runtime data.
+#
+# Copyright (C) 2025 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 31 Milk Street #960789 Boston, MA
+# 02196 USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+from dasbus.structure import DBusData
+from dasbus.typing import *  # pylint: disable=wildcard-import
```

Comment:
Just a note: The plan is to replace these wildcard imports in the future with this:
``` python
from dasbus import typing as dt
```

and then we can modify the code that uses types like this:

``` python
   @property
    def enabled(self) -> dt.Bool:
```

I don't think it's worthwhile to make this change to this PR.  We can start with new code and then fix all of the old places as a cleanup task.